### PR TITLE
Add validators for some fields

### PR DESCRIPTION
**Pull request by alrod**

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -36,7 +36,13 @@
                     "defaultValue": "samples-workitems",
                     "required": true,
                     "label": "Event Hub name",
-                    "help": "This is the name of the Event Hub you would like to trigger off of."
+                    "help": "This is the name of the Event Hub you would like to trigger off of.",
+                    "validators": [
+                        {
+                            "expression": "^[A-Za-z0-9]$|^[A-Za-z0-9][\\w-\\.\\/]*[A-Za-z0-9]$",
+                            "errorText": "Not valid event hub name"
```
Comment (soninaren) [inline Bindings/bindings.json]: Need to escape the \ in the regex

File: Bindings/bindings.json [hunk 2]
```diff
@@ -136,7 +160,13 @@
                     "defaultValue": "samples-workitems/{name}",
                     "required": true,
                     "label": "Path",
-                    "help": "This is the path within your storage account to which the blob will be written."
+                    "help": "This is the path within your storage account to which the blob will be written.",
+                    "validators": [
+                        {
+                            "expression": "^([a-z0-9](([a-z0-9\-[^\-])){1,61}[a-z0-9])(\/)(\S){1,1023}$",
+                            "errorText": "Not valid blob path"
```
Comment (soninaren) [inline Bindings/bindings.json]: '\' is not escaped here

File: Bindings/bindings.json [hunk 3]
```diff
@@ -186,7 +222,13 @@
                     "defaultValue": "samples-workitems",
                     "required": true,
                     "label": "Path",
-                    "help": "This is the path within your storage account that the trigger will monitor."
+                    "help": "This is the path within your storage account that the trigger will monitor.",
+                    "validators": [
+                        {
+                            "expression": "^([a-z0-9](([a-z0-9\-[^\-])){1,61}[a-z0-9])(\/)(\S){1,1023}$",
+                            "errorText": "Not valid blob path"
```
Comment (soninaren) [inline Bindings/bindings.json]: '\' is not escaped here

File: Bindings/bindings.json [hunk 4]
```diff
@@ -161,7 +191,13 @@
                     "defaultValue": "samples-workitems/{name}",
                     "required": true,
                     "label": "Path",
-                    "help": "This is the path within your storage account from which the blob will be read."
+                    "help": "This is the path within your storage account from which the blob will be read.",
+                    "validators": [
+                        {
+                            "expression": "^([a-z0-9](([a-z0-9\-[^\-])){1,61}[a-z0-9])(\/)(\S){1,1023}$",
```
Comment (soninaren) [inline Bindings/bindings.json]: '\' is not escaped here

Comment (alrod) [inline Bindings/bindings.json]: Fixed

**Comment by soninaren** [comment]
looks good :+1:

**Pull request closed without merging**